insights/client/apps/malware_detection/__init__.py

```diff
@@ -796,7 +802,13 @@ def scan_filesystem(self):
                 logger.debug("Unable to scan %s: %s", toplevel_dir, cpe.output.strip())
                 continue
 
-            self.parse_scan_output(output.strip())
+            try:
+                self.parse_scan_output(output.strip())
+            except Exception as e:
+                self.potential_matches += 1
```

Comment:
I'm curious what the potential_matches is used for, I see it was added in this PR but isn't used by anything added in the PR? Unless I missed something.

Reply:
@ryan-blakley well, if we hit an exception in that code we are in processing the yara output, so it means we had rule matches but we won't know exactly what they were because of the exception.  So potential_matches is just used to triggger a message to the user that although we don't have any specific scan matches to report to you, we potentially found matches.  And you should probably contact support to help troubleshoot the problem further.

Anyway, that's the reasoning behind potential_matches.

Reply:
Okay thanks for explaining, I just wanted to make sure it wasn't something that was meant to be used but got removed.